src/pill/index.html

```diff
@@ -0,0 +1,8 @@
+<div class="gb-pill-container { gb-pill-component-group: props.closeaction }">
+  <a class="gb-pill-component pill { gb-pill-selected: props.selected }" onclick="{ onClick }">
+    <yield/>
+  </a>
+  <a if="{ props.closeaction }" class="gb-pill-component pill { gb-pill-close-selected: props.selected }" onclick="{ onClose }">
```

Comment:
closeAction should be camelcase. In html only on the left hand side of the equation needs to be all lower case

Reply:
"gb-pill-close-selected": without knowing it really does, I may suggest  something like "inactive-seleted"